CPC-CHARTER.md

```diff
@@ -233,51 +233,34 @@ election is required if there is only one candidate and no objections to
 the candidates election. Elections shall be done within the Projects by
 the Collaborators active in the Project.
 
+The CPC will elect from amongst CPC members:
 
-The CPC will elect from amongst CPC members a CPC Chairperson to
-work on building an agenda for CPC meetings and a CPC Director to represent
-the CPC to the Board for a term of one year according to the OpenJS Foundation’s
-By-laws. The Chair and Director may be (but are not required to be)
-the same person. The CPC shall hold annual elections to select a CPC
-Chairperson and Director; there are no limits on the number of terms a
-CPC Chairperson or Director may serve.
+* a CPC Chairperson to work on building an agenda for CPC meetings
+* CPC Directors to represent the Foundation's projects and related communities to the Board
 
-## Section 8. Board Representation
+The term for each of these roles is one year (as defined in the [OpenJS Foundation bylaws][]).
 
-There are 3 Board seats earmarked for representation from the
-OpenJS Foundation, the Foundation's projects, and related communities.
+The CPC shall hold annual elections to select a CPC Chairperson and Directors.
+The CPC can decide to spread those elections across the year
+in order to avoid changing all of its Board representatives at once.
```

Comment:
nit, for better diffs:
```suggestion
The CPC can decide to spread those elections across the year in order to avoid changing all of its Board representatives at once.
```